### PR TITLE
Guideline to install JTL reporter on Azure

### PR DESCRIPTION
**Pull request by C0DINGpanda**
Guideline to install JTL reporter on Azure

### CODE REVIEW

**Comment by C0DINGpanda** [comment]
@ludeknovy @dennisconrad could u please check and approve this also i got no access for preview and apreview build is failing.

**Comment by ludeknovy** [comment]
@esahanhunter Thanks for this PR. Please could you check the failing build first? 
```
syntaxError: /vercel/path0/docs/guides/Azure-Installation.md: Expected corresponding JSX closing tag for <postgresDB> (34:59)
```

**Comment by C0DINGpanda** [comment]
@ludeknovy fixed

File: docs/guides/Azure-Installation.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+you need to create the following azure resources before u begin:
+* Azure Resorce group.
+* Azure storage account with 2 file shares (for mongo and Postgres databases).
+* Azure storage account access key.
+
+#### Instalation steps: 
+1. Create Azure resource group 
+2. Create azure storage account with 2 file shares.
+3. Get the storage account access key.
+4. clone the [git repo](https://github.com/esahanhunter/AzureJTLReporter).
```
Comment (ludeknovy) [inline docs/guides/Azure-Installation.md]: Please make the first letter capital

File: docs/guides/Azure-Installation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+5. download and install the latest version of [PSQLODBC](https://www.postgresql.org/ftp/odbc/versions/msi/) to run psql commands.
```
Comment (ludeknovy) [inline docs/guides/Azure-Installation.md]: Please make the first letter capital

File: docs/guides/Azure-Installation.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        $ResourceGroup = "<ResourceGroup Name>"
+        $Storageaccountname = "<Storage account name>"
+        $storageKey = "<Storage key>"
+        $mongodbshareName = "<MongoDB Fileshare name >"
+        $postgresdbFileshare = "<postgresDB Fileshare name >"
+    ```
+        
+7. Connect to Azure using powershell and run DeployJTLReport.ps1
+8. Post success you can delete the Migration container. 
+9. You can access JWT report  using the IP of jtlreportr
```
Comment (ludeknovy) [inline docs/guides/Azure-Installation.md]: Suggestion: `JtlReporter should be running at:` ?

File: docs/guides/Azure-Installation.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    ```IP
+        http://<ContainerIP>:80
```
Comment (ludeknovy) [inline docs/guides/Azure-Installation.md]: Did you remap the default docker port to :80?

File: docs/guides/Azure-Installation.md (hunk 1 above)
Comment (ludeknovy) [reply docs/guides/Azure-Installation.md]: Would you be willing to move the deployment scripts into https://github.com/ludeknovy/jtl-reporter ?

File: docs/guides/Azure-Installation.md (hunk 4 above)
Comment (C0DINGpanda) [reply docs/guides/Azure-Installation.md]: Port 80 and 2020 both are exposed in 'fe' they should work either ways.

File: docs/guides/Azure-Installation.md (hunk 1 above)
Comment (C0DINGpanda) [reply docs/guides/Azure-Installation.md]: updated the typos and have created the PR to push the scripts. :)

File: docs/guides/Azure-Installation.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        $Storageaccountname = "<Storage account name>"
+        $storageKey = "<Storage key>"
+        $mongodbshareName = "<MongoDB File share name >"
+        $postgresdbFileshare = "<PostgresDB File share name >"
+    ```
+        
+7. Connect to Azure using PowerShell and run DeployJTLReport.ps1
+8. After script execution you can see 3 container instances in your Azure resource group (JtlReporter, postgres and Migration).
+9. Post success you can delete the Migration container. 
+9. You can access JWT report  using the IP of JtlReporter container instance.
```
Comment (ludeknovy) [inline docs/guides/Azure-Installation.md]: This still needs some polishment I think... Whats `JWT report` ?

**Comment by ludeknovy** [comment]
@esahanhunter thanks for your contribution!

**Pull request merged**